packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -56,7 +56,7 @@ class SelectScsiId extends Component {
     () => this.props.options,
     options =>
       map(options, ({ vendor, path, size, scsiId }) => ({
-        label: `${vendor} - ${path} (${formatSize(size)})`,
+        label: `${vendor} - ${path} (${formatSize(+size)})`,
```

Comment:
@pdonias Maybe this should be fixed on server's side instead?

Reply:
I have no experience on that side, but I'll take a look and see if it can be done. Any help in this is appreciated.

Reply:
No problem!
`size: hbaDevice.size.trim()` → `size: +hbaDevice.size.trim()`
and you can remove the change in `xo-web/src/xo-app/new/sr/index.js `

Reply:
Just do the same change you did on the UI but in front of `hbaDevice.size.trim()`.

If you don't want/can't do it, no problem, we do it on our side :slightly_smiling_face:

Reply:
I moved the fix to server. Thanks for the help :)